component-playground/lib/get-component-fixture-tree.js

```diff
@@ -1,13 +1,15 @@
 var parseFixturePath = require('./parse-fixture-path.js');
+var parseComponentPath = require('./parse-component-path.js');
 
 module.exports = function() {
   var requireFixture = require.context('fixtures', true, /\.js$/),
-      fixtures = {};
+    requireComponent = require.context('components', true, /\.jsx?$/),
+    fixtures = {};
 
   requireFixture.keys().forEach(function(fixturePath) {
     var pathParts = parseFixturePath(fixturePath),
-        componentName = pathParts[1],
-        fixtureName = pathParts[2];
+      componentName = pathParts[1],
+      fixtureName = pathParts[2];
```

Comment:
Why re-indent these lines? I prefer 4 line indenting when aligning multiline declarations or calls. It's part of the Google JS style guide that I follow. This applies for the lines you added above as well.

PS. I'm in the process of composing a contributing guide to make the PR process smoother.